eventually/tests/unittests/validators/testmodel.py

```diff
@@ -0,0 +1,97 @@
+"""Validators Test."""
```

Comment:
Why validators/testmodel.py?. We don't test models in validators. Maybe better validators/test.py or validators/testfunctions.py

Reply:
done